### PR TITLE
zh-translation: docs/setup/platform-setup/kops/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/platform-setup/kops/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: Kops
+description: 与Istio 一起使用的 Kops 设置说明。
+weight: 33
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kops]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+如果您想要在 Kops 管理的集群上为 Mesh 运行 Istio [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#sds-configuration) (SDS)，必须添加 [Extra Configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)，以便在 Api-Server 中启动服务令牌 Projection Volumes。
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/kops/index.md:11]:
```suggestion
如果您想要在 Kops 管理的集群上为 Mesh 运行 Istio [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#sds-configuration) (SDS)，必须添加 [Extra Configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)，以便在 API Server 中启动服务令牌 Projection Volumes。
```

**Comment by orangegzx** [comment]
modified

**Review by kebe7jun — APPROVED** [review]
lgtm

File: content/zh/docs/setup/platform-setup/kops/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: Kops
+description: 与Istio 一起使用的 Kops 设置说明。
+weight: 33
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kops]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+如果您想要在 Kops 管理的集群上为 Mesh 运行 Istio [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#sds-configuration) (SDS)，必须添加 [Extra Configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)，以便在 Api Server 中启动服务令牌 Projection Volumes。
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/kops/index.md:11]:
```suggestion
如果您想要在 Kops 管理的集群上为 Mesh 运行 Istio [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#sds-configuration) (SDS)，必须添加 [Extra Configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)，以便在 API Server 中启动服务令牌 Projection Volumes。
```

**Pull request merged**